www/src/components/layout/content-container.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { css } from "react-emotion"
+
+import presets from "../../utils/presets"
+
+import { rhythm, options } from "../../utils/typography"
+
+const contentContainer = css`
+  background: red;
+  position: relative;
+`
```

Comment:
In this case, it's preferable to use `styled('div')` to clean up the component definition, e.g. we can scan for `<ContentContainer>{/*...*/</ContentContainer>` vs. `<div ...>{/*...*/}</div>`.

Reply:
As above, also in my next PR I will unify all Emotion components to one syntax, 
`styled("div")`
In the PR I use also `styled.div(` and that's inconsistency